### DISCUSSION TITLE
[BUG] Lost coin

### DISCUSSION THREAD

**Discussion by zoquile19901228** [Chia-Network/chia-blockchain#5317] [Support]
I can still view the unsettled rewards on May 15, 2021, but when I go online at 4:00 p.m. on May 16, 2021, I find that the income is gone. May I ask if the account is blocked or displayed incorrectly? Account number: 418094790@qq.com

Block coverage: 283550，283520，283500，283430

**Comment by nuistzj2014** [comment]
the same as you , you can search whether having transactions info or not!

**Comment by zoquile19901228** [comment]
> 和你一样，你可以搜索是否有交易信息！
There is no transaction information.

**Reply by nuistzj2014** [reply]
I have two txs which I have never send, that caused  6 xch lost

**Reply by nuistzj2014** [reply]
And the transaction value does not match the lost amount

**Reply by zoquile19901228** [reply]
Please ask the official to give me an explanation

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
